### ISSUE TITLE
The btrfs volume manager is not working correctly

### ISSUE THREAD

**Issue by jloehel** [SUSE/kiwi#910]
## Problem description
kiwi throws an exception while getting the sysconfig file for snapper.
```
[  219s] [ ERROR   ]: 13:06:39 | Unexpected error:
[  219s] Traceback (most recent call last):
[  219s]   File "/usr/bin/kiwi", line 11, in <module>
[  219s]     load_entry_point('kiwi==9.17.1', 'console_scripts', 'kiwi-ng-3')()
[  219s]   File "/usr/lib/python3.6/site-packages/kiwi/kiwi.py", line 64, in main
[  219s]     App()
[  219s]   File "/usr/lib/python3.6/site-packages/kiwi/app.py", line 35, in __init__
[  219s]     task_class().process()
[  219s]   File "/usr/lib/python3.6/site-packages/kiwi/tasks/system_build.py", line 269, in process
[  219s]     result = image_builder.create()
[  219s]   File "/usr/lib/python3.6/site-packages/kiwi/builder/disk.py", line 170, in create
[  219s]     result = disk.create_disk()
[  219s]   File "/usr/lib/python3.6/site-packages/kiwi/builder/disk.py", line 398, in create_disk
[  219s]     self._get_exclude_list_for_root_data_sync(device_map)
[  219s]   File "/usr/lib/python3.6/site-packages/kiwi/volume_manager/btrfs.py", line 315, in sync_data
[  219s]     self._create_snapper_quota_configuration()
[  219s]   File "/usr/lib/python3.6/site-packages/kiwi/volume_manager/btrfs.py", line 371, in _create_snapper_quota_configuration
[  219s]     config_file = self._set_snapper_sysconfig_file(root_path)
[  219s]   File "/usr/lib/python3.6/site-packages/kiwi/volume_manager/btrfs.py", line 384, in _set_snapper_sysconfig_file
[  219s]     if not sysconf_file.get('SNAPPER_CONFIGS') or \
[  219s] AttributeError: 'SysConfig' object has no attribute 'get'
```

## Expected behaviour
The check if the config file is present returns true or false.

## Steps to reproduce the behaviour
All the image builds in devel:kubic:images:experimental on OBS are failing.

## OS and Software information
+ KIWI version: python3-kiwi-9.17.1-1.1
+ Operating system: openSUSE Tumbleweed
+ OBS version: Head

**Comment by schaefi** [comment]
Indeed, wondering why we merged it :(

@davidcassany the implementation of the SysConfig class is not suitable for use with the
standard ```get``` method. See the below example which leads to the bug reported here.

Could you fix this one ? Thanks

```python
class Foo:
   def __init__(self):
       self.data_dict = {
           'foo': 'bar'
       }

   def __getitem__(self, key):
        return self.data_dict[key]

x = Foo()

print(x['foo'])

print(x.get('foo'))
```

**Comment by schaefi** [comment]
@davidcassany nevermind I needed the fix for a test, so did it in #912
